fluentui_tablayout/src/main/java/com/microsoft/fluentui/tokenized/navigation/TabBar.kt

```diff
@@ -59,7 +60,7 @@ fun TabBar(
         )
         Row(
             modifier = Modifier
-                .padding(horizontal = 16.dp)
+                .padding(horizontal = if(LocalConfiguration.current.screenWidthDp.dp > 360.dp )16.dp else 1.dp)       // For small phones we don't want the text overflowing, hence removing padding
```

Comment:
What if the device is small and the number of items are 2. There will be no padding and is this expected? Should we removed the padding only when the items are more?

Reply:
![small phone](https://github.com/microsoft/fluentui-android/assets/68989156/8d35106e-25ab-4055-9968-0b1525ef7113)
There's plenty of space in less number of items